main.py

```diff
@@ -1,10 +1,44 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player in data:
+        race_name = data[player]["race"]["name"]
+        race_description = data[player]["race"]["description"]
+        Race.objects.get_or_create(name=race_name,
+                                   description=race_description)
+
+        for skill in data[player]["race"]["skills"]:
+            skill_name = skill["name"]
+            skill_bonus = skill["bonus"]
+            Skill.objects.get_or_create(name=skill_name,
+                                        bonus=skill_bonus,
+                                        race=Race.objects.get(name=race_name))
```

Comment:
This is a performance issue. You are making a database query inside a loop. This could lead to a large number of queries if there are many players and each player has many skills, which could slow down your program. Consider using `bulk_create` to create multiple objects in one query. However, you will need to handle the case where the skill already exists in the database yourself.